_src/pattern-library/sass/components/_copy.scss

```diff
@@ -108,4 +108,13 @@
     @extend %copy-micro;
 }
 
+// broad stroke extend for emphasized/de-emphasized copy
+%copy-emphasized {
```

Comment:
Awesome addition! This will come in handy. Any chance I can ask for a further refactor that includes the following?

1. Define this near the top of the file (within the ``#GLOBAL``) section 
2. Use these extends to replace some of the [canned class rules here](https://github.com/edx/ux-pattern-library/blob/frances/emphasized-extends/_src/pattern-library/sass/components/_copy.scss#L15)